custom_model_runner/datarobot_drum/resource/components/Python/uwsgi_component/uwsgi_serving.py

```diff
@@ -173,6 +173,12 @@ def prediction_server_stats(self, url_params, form_params):
     def predictions(self, url_params, form_params):
         return self.predict(url_params, form_params)
 
+    @FlaskRoute(
+        "{}/invocations".format(os.environ.get(URL_PREFIX_ENV_VAR_NAME, "")), methods=["POST"]
+    )
+    def predictions(self, url_params, form_params):
```

Comment:
Looks like tests fail just because of this:
17:23:47  2023-11-29 01:20:43,136 INFO [uwsgi_broker:314]:  Add routing endpoint: ('/health/', None, 'health', {'methods': ['GET']}, False)
17:23:47  2023-11-29 01:20:43,137 INFO [uwsgi_broker:314]:  Add routing endpoint: ('/stats/', None, 'prediction_server_stats', {'methods': ['GET']}, False)
17:23:47  2023-11-29 01:20:43,137 INFO [uwsgi_broker:314]:  Add routing endpoint: ('/predictions/', None, 'predictions', {'methods': ['POST']}, False)
17:23:47  2023-11-29 01:20:43,138 INFO [uwsgi_broker:314]:  Add routing endpoint: ('/invocations', None, 'predictions', {'methods': ['POST']}, False)
17:23:47  Traceback (most recent call last):
17:23:47    File "uwsgi_entry_script.py", line 15, in <module>
17:23:47      UwsgiBroker.uwsgi_entry_point(
17:23:47    File "/usr/local/lib/python3.9/dist-packages/mlpiper/components/restful/uwsgi_broker.py", line 296, in uwsgi_entry_point
17:23:47      cls._setup_flask_app(pipeline_name, cls.w_logger)
17:23:47    File "/usr/local/lib/python3.9/dist-packages/mlpiper/components/restful/uwsgi_broker.py", line 315, in _setup_flask_app
17:23:47      app.add_endpoint(
17:23:47    File "/usr/local/lib/python3.9/dist-packages/mlpiper/components/restful/flask_app_wrapper.py", line 77, in add_endpoint
17:23:47      FlaskAppWrapper.app.add_url_rule(
17:23:47    File "/usr/local/lib/python3.9/dist-packages/flask/scaffold.py", line 50, in wrapper_func
17:23:47      return f(self, *args, **kwargs)
17:23:47    File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1361, in add_url_rule
17:23:47      raise AssertionError(
17:23:47  AssertionError: View function mapping is overwriting an existing endpoint function: predictions

Reply:
Done